### PR TITLE
fix: updated the aragorn 

### PR DESCRIPTION
**Pull request by chevvak2**

### CODE REVIEW

File: helm/aragorn/values.yaml [hunk 1]
```diff
@@ -116,11 +116,15 @@ redis:
   image:
     repository: redis
     tag: "7.0.12"
+  replicaCount: 2
```
Comment (YaphetKG) [inline helm/aragorn/values.yaml:119]: is there a reason why we have 2 replicas?

File: helm/aragorn/values.yaml [hunk 2]
```diff
@@ -116,11 +116,15 @@ redis:
...
   master:
     resources:
       limits:
-        cpu: 2
-        memory: 60Gi
+        cpu: 8
```
Comment (YaphetKG) [inline helm/aragorn/values.yaml:123]: why lower the memory than CI?

File: helm/aragorn/values.yaml (hunk 1 above)
Comment (chevvak2) [reply helm/aragorn/values.yaml:119]: Oh I have removed it, let me update it

File: jenkins/aragorn/Jenkinsfile [hunk 3]
```diff
@@ -121,6 +121,8 @@ pipeline {
                         ]) {
                             sh '''
                             aws --region ${AWS_REGION} eks update-kubeconfig --name ${KUBERNETES_CLUSTER_NAME}
+                            helm delete $APP -n sri 
+                            kubectl delete pvc redis-data-aragorn-redis-master-0 -n sri
```
Comment (maximusunc) [inline jenkins/aragorn/Jenkinsfile:125]: Will this delete the existing PVC when we do a new deployment? I'm fairly sure we don't want this.

File: helm/aragorn/values.yaml (hunk 2 above)
Comment (chevvak2) [reply helm/aragorn/values.yaml:122]: If we give 60 the application goes into a pending state to create PVC, so I lower the memory to use 58G. If anything breaks the application will try to fix it.

**Review by YaphetKG — APPROVED** [review]

**Pull request merged**